internal/controller/sonataflow.go

```diff
@@ -191,25 +260,32 @@ func getSonataFlowPlatformSpec(orchestrator *orchestratorv1alpha1.Orchestrator)
 		Services: &sonataapi.ServicesPlatformSpec{
 			DataIndex: &sonataapi.ServiceSpec{
 				Enabled:     util.MakePointer(true),
-				Persistence: getSonataFlowPersistence(orchestrator),
+				Persistence: getServerlessLogicPersistence(orchestrator),
 			},
 			JobService: &sonataapi.ServiceSpec{
 				Enabled:     util.MakePointer(true),
-				Persistence: getSonataFlowPersistence(orchestrator),
+				Persistence: getServerlessLogicPersistence(orchestrator),
 			},
 		},
 	}
 }
 
-func handleSonataFlowCleanUp(ctx context.Context, client client.Client, olmClientSet olmclientset.Clientset) error {
+func handleServerlessLogicCleanUp(ctx context.Context, client client.Client, olmClientSet olmclientset.Clientset, namespace string) error {
```

Comment:
How do we confirm a disabled subscription (when the user set "disable" for the install operator in the Orchestrator CR), that was installed by the admin, isn't removed by this logic?
Don't we need to check a specific annotation or label to guarantee we remove only resources created by the Orchestrator operator?

Reply:
Yes, we will need to check the _created-by=orchestrator_ label we are adding to the resources handled by orchestrator before deleting, otherwise this will remove and we don't want that. Currently, we are checking the label in RHDH CR before deleting. Will review the cleanup process and add these checks for the resources to ensure proper deletion.
Created the ticket [FLPATH-1911](https://issues.redhat.com/browse/FLPATH-1911) so i don't miss it
Will also update the ADR with this detail.